core/src/main/java/org/jdbi/v3/core/mapper/reflect/ReflectionMappers.java

```diff
@@ -30,6 +32,7 @@ public class ReflectionMappers implements JdbiConfig<ReflectionMappers> {
     private List<ColumnNameMatcher> columnNameMatchers;
     private boolean strictMatching;
     private UnaryOperator<String> caseChange;
+    private Consumer<AccessibleObject> makeAccessible = accessibleObject -> accessibleObject.setAccessible(true);
```

Comment:
"accessibleStrategy"

Reply:
this introduces a bug. If someone *really* bothers to register a strategy, then after the first copy operation (which calls `new ReflectionMappers(oldReflectionMappers)`, the strategy will be reset to the default because the field was not added to the copy constructor.